README.md

```diff
@@ -16,6 +16,9 @@ For cyclical learning rates (also detailed in Leslie Smith's paper) where the le
 Python 2.7 and above:
 ``pip install torch-lr-finder``
 
+Install with the support of mixed precision training:
+``pip install torch-lr-finder -v --global-option="amp"``
+
```

Comment:
Apex requires Python 3, so we should state that, or tell the reader to check Apex requirements and link to the repo

Reply:
I'm with you on that. And I'll add a simple check in `setup.py` to check that, too.